src/game/Game.ts

```diff
@@ -0,0 +1,128 @@
+import KeyListener from "./KeyListener";
+
+enum Tile {
+    FLOOR,
+    WALL,
+    UNDEFINED,
+}
+
+interface PlayerObject {
+    r: number;
+    c: number;
+}
+
+export default class Game {
+    map: Tile[][];
```

Comment:
A class for a `Map` might be useful too. If we keep track of the entities that exist in a Map we can easily query for the win condition (number of monsters alive == 0). 

If we have separate layers for things (Tiles under Entities) this will help encapsulate things too.